program/tests/duplicate_block_proof.rs

```diff
@@ -124,6 +135,58 @@ async fn write_proof(
     }
 }
 
+async fn close_report(
+    context: &mut ProgramTestContext,
+    report_key: Pubkey,
+    destination: Pubkey,
+    initial_lamports: u64,
```

Comment:
Up to you, but rather than passing this in, it could be fetched from the report account in this function

Reply:
Saves us an arg, fetched it instead 68e2d6a16f8e6aae161e633aee3954abfbbf046d